net/instaweb/rewriter/css_filter.cc

```diff
@@ -73,13 +73,6 @@
 #include "pagespeed/opt/logging/enums.pb.h"
 #include "webutil/css/parser.h"
 
-#include "base/at_exit.h"
-
-namespace {
-
-base::AtExitManager* at_exit_manager = NULL;
```

Comment:
@morlovich Sorry, late reply. 
Do you mean to ask why this is deleted here? I had to move this initialization over to a more central location because it has to be performed exactly once. Its new home ended up being `process_context.cc` (see later on in this change). 
Also see the changes in `http_names.cc`, which add another dependency by using`LazyInstance`.